Shah docs

ShahFragcolor opened the pull request:
<!--
Before submitting your PR, please review the following checklist:

- [ ] **DO NOT** submit a PR without having discussed the change first in a related issue. If none exist, create one first.
- [ ] **CONSIDER** adding a unit test if your PR resolves an issue.
- [ ] **DO** keep pull requests small so they can be easily reviewed.
- [ ] **DO** make sure all unit tests pass.
- [ ] **DO** make sure not to introduce any new compiler warnings.
- [ ] **AVOID** breaking the continuous integration build.
- [ ] **AVOID** making significant changes to the overall architecture.
-->

ShahFragcolor committed:
Documented wires.cpp and wires.hpp

- added input and output help text for all wire executors
- edited help text for all wire executors
- edited the parameter text for all wire executors

sinkingsugar commented on shards/modules/core/wires.cpp:
```diff
@@ -244,9 +244,18 @@ struct Wait : public WireBase {
     passthrough = false; // also need this to have proper compose output type
   }
 
+  static SHOptionalString inputHelp() {
```
In the case of passthru we should not repeat the string all the time, we should have a string declared in a common place and always reuse that for consistance and less memory usage

sinkingsugar replied:
```
struct CoreInfo {
  static inline SHOptionalString PTInputHelp = SHCCSTR("Passthru input");
  static inline SHOptionalString PTOutputHelp = SHCCSTR("Passthru input")
  ```

sinkingsugar commented on shards/modules/core/wires.cpp:
```diff
@@ -357,11 +366,22 @@ struct IsRunning : public WireBase {
   }
 
-  SHOptionalString help() { return SHCCSTR("Checks if a wire is running and outputs true if that is the case, false if not."); }
+  static SHOptionalString inputHelp() { return SHCCSTR("The input of this shard is ignored."); }
```
Standardize this too

sinkingsugar commented on shards/modules/core/wires.cpp:
```diff
@@ -1399,9 +1477,9 @@ struct ParallelBase : public CapturingSpawners {
-      {"Wire", SHCCSTR("The wire to spawn and try to run many times concurrently."), IntoWire::RunnableTypes},
-      {"Policy", SHCCSTR("The execution policy in terms of wires success."), {WaitUntilEnumInfo::Type}},
-      {"Threads", SHCCSTR("The number of cpu threads to use."), {CoreInfo::IntType}}};
+      {"Wire", SHCCSTR("The Wire to copy and schedule."), IntoWire::RunnableTypes},
```
I noticed above too, not sure we need to capitalize those but if helps clarity why not

ShahFragcolor committed:
Standardized some input and output helps

- created a const in common_types and wries.hpp to standardize common input texts and output texts

ShahFragcolor committed:
Shifted the default help

Shifted the default help text from common_types.cpp to runtime.hpp
